Solutions/Infoblox Cloud Data Connector/Analytic Rules/Infoblox-HighNumberOfHighThreatLevelQueriesDetected.yaml

```diff
@@ -0,0 +1,42 @@
+id: 57113ad7-7dd6-4150-84d8-252e162aaf4a
+name: Infoblox - High Number of High Threat Level Queries Detected
+description: |
+  'This creates an incident in the event a single host generates a high number of high threat level queries.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: InfobloxCloudDataConnector
+    dataTypes: 
+      - CommonSecurityLog (InfobloxCDC)
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Impact
+relevantTechniques:
+  - T1498
+  - T1565
+query: |
+
+  let threshold = 200;
+  InfobloxCDC
+  | where DeviceEventClassID has_cs "RPZ"
```

Comment:
Please add timefilter in all queries. 

`| where TimeGeenrated > ago(1d)`

Reply:
Hi @ashwin-patil, Wouldn't this potentially miss data if the user edited the rule period to be bigger than the TimeGenerated? I.e If the TimeGenerated returns results from 1 hour ago but the rule period is 2 hours, it would skip 1 hour of data. Is that right?